indexer/indexers.py

```diff
@@ -40,9 +40,17 @@ class Indexer:
     """
 
     def __init__(self):
-        self.connection = connections.create_connection(
-            hosts=settings.ELASTICSEARCH['default']['hosts'], timeout=60
-        )
+        if settings.ELASTICSEARCH['default']['api_key']:
```

Comment:
I'm thinking a better way to do this would be to put the construction of the client args in the if/else statements, and then pass those to the create_connection function. Something like:

```
connection_args = {'hosts': settings.ELASTICSEARCH['default']['hosts'], 'timeout': 60}
if settings.ELASTICSEARCH['default'].get('api_key'):
    connection_args['api_key'] = settings.ELASTICSEARCH['default']['api_key']
self.connection = connections.create_connection(**connection_args)
```